api/tests/routes/backoffice/providers_test.py

```diff
@@ -26,36 +26,55 @@
 ]
 
 
-class GetProvidersPageTest(GetEndpointHelper):
-    endpoint = "backoffice_web.providers.get_providers"
+class ListProvidersPageTest(GetEndpointHelper):
+    endpoint = "backoffice_web.providers.list_providers"
     needed_permission = perm_models.Permissions.MANAGE_TECH_PARTNERS
 
     # - fetch session (1 query)
     # - fetch user (1 query)
     # - fetch providers and associated api keys (1 query)
     expected_num_queries = 3
 
-    def test_get_providers(self, authenticated_client):
-        offerer = offerers_factories.OffererFactory(name="Individual Offer API consumer")
+    def test_list_providers(self, authenticated_client):
+        offerer = offerers_factories.OffererFactory(name="Aaaaaah je suis au début")
         provider = providers_factories.ProviderFactory(name=offerer.name)
         providers_factories.OffererProviderFactory(offerer=offerer, provider=provider)
         offerers_factories.ApiKeyFactory(offerer=offerer, provider=provider)
 
+        second_offerer = offerers_factories.OffererFactory(name="Zzzzz je dors à la fin")
+        second_provider = providers_factories.ProviderFactory(name=second_offerer.name)
+        providers_factories.OffererProviderFactory(offerer=second_offerer, provider=second_provider)
+        offerers_factories.ApiKeyFactory(offerer=second_offerer, provider=second_provider, prefix="other-prefix")
+        providers_factories.VenueProviderFactory(provider=second_provider)
+        providers_factories.VenueProviderFactory(provider=second_provider, isActive=False)
+
         with assert_num_queries(self.expected_num_queries):
             response = authenticated_client.get(url_for(self.endpoint))
             assert response.status_code == 200
 
-        provider_rows = html_parser.extract_table_rows(response.data)
-
-        provider_row = next((row for row in provider_rows if row["ID"] == str(provider.id)))
-        assert provider_row["Partenaire technique"] == provider.name
-        assert provider_row["SIREN"] == offerer.siren
-        assert provider_row["Ville"] == offerer.city
-        assert provider_row["Code postal"] == offerer.postalCode
-        assert provider_row["URL du logo"] == ""
-        assert provider_row["Nombre de clés d'API"] == str(1)
-        assert provider_row["Actif pour les pros"] == "Oui"
-        assert provider_row["Actif"] == "Oui"
+        rows = html_parser.extract_table_rows(response.data)
+
+        assert rows[0]["ID"] == str(provider.id)
+        assert rows[0]["Partenaire technique"] == provider.name
+        assert rows[0]["Lieux synchronisés au partenaire"] == "0 actif / 0 inactif"
+        assert rows[0]["SIREN"] == offerer.siren
+        assert rows[0]["Ville"] == offerer.city
+        assert rows[0]["Code postal"] == offerer.postalCode
+        assert rows[0]["URL du logo"] == ""
+        assert rows[0]["Nombre de clés d'API"] == str(1)
```

Comment:
Anecdotique : de tête, ça devrait souvent donner : 
```suggestion
        assert rows[0]["Nombre de clés d'API"] == "1"
```